frontend/app/assets/javascripts/spree/frontend/views/spree/products/cart_form.js

```diff
@@ -243,21 +243,24 @@ Spree.ready(function($) {
     )
   }
 
+  Spree.addToCartFormSubmissionOptions = function() {
+    return {};
+  }
+
   $('#product-details').on('submit', ADD_TO_CART_FORM_SELECTOR, function(event) {
-    var variantId
-    var quantity
-    var $cartForm = $(event.currentTarget)
-    var $addToCart = $cartForm.find(ADD_TO_CART_SELECTOR)
+    let $cartForm = $(event.currentTarget);
+    let $addToCart = $cartForm.find(ADD_TO_CART_SELECTOR);
+    let variantId = $cartForm.find(VARIANT_ID_SELECTOR).val();
+    let quantity = parseInt($cartForm.find('[name="quantity"]').val());
+    let options = Spree.addToCartFormSubmissionOptions();
```

Comment:
@rishabhguptarishi, unfortunately, we cannot use ES6 syntax yet, please use `var` keyword

Reply:
@damianlegawiec done :thumbsup: